res/css/views/audio_messages/_AudioPlayer.scss

```diff
@@ -58,10 +58,10 @@ limitations under the License.
         }
 
         .mx_Clock {
-            width: $font-42px; // we're not using a monospace font, so fake it
```

Comment:
does this now make the width bounce as the time changes?

Reply:
I don't think its about the position of the `:` but more about the width changing and thus any text it causes to overflow to its left also moving

Reply:
Since this is a child element of flexbox and `min-width` is defined below, overflow should not happen. The actual width change of the counter is minimized with `text-align: justify` as well.

Unless you would play a *very* long audio file more than a day (01:00:00:00) on a fairly small window, the seek bar should also be available :-D

Reply:
@t3chguy Would you please point me out what to be addressed? I am not really sure about it.. thanks!